guide/interactions/modals.md

```diff
@@ -0,0 +1,128 @@
+# Modals
+
+With modals, you can create pop-up forms that allow users to provide you, with formatted inputs, through submissions. We'll cover how to create, show, and receive modal forms using discord.js!
+
+::: tip
+This page is a follow-up to the [interactions (slash commands) page](/interactions/slash-commands.md). Please carefully read those first so that you can understand the methods used in this section.
```

Comment:
```suggestion
This page is a follow-up to the [interactions (slash commands) page](/interactions/slash-commands.md). Please carefully read that section first, so that you can understand the methods used in this section.
```